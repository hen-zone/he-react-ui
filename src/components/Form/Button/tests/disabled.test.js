import createTestContext from 'react-cosmos-test/enzyme'
import fixture from '../fixtures/disabled.fixture'

const { mount, getWrapper } = createTestContext({ fixture })

beforeEach(mount)

test('<Button /> rendered correctly with disabled fixture', () => {
  let wrapper = getWrapper()
  expect(wrapper.find('button')).toHaveLength(1)

  expect(wrapper).toMatchSnapshot()
})
