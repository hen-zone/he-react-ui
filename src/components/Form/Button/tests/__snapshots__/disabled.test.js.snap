// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> rendered correctly with disabled fixture 1`] = `
<Button
  color="teal"
  disabled={true}
  icon=""
  keyline={false}
  link={false}
  onClick={[Function]}
  squared={false}
  submit={false}
  withRouter={false}
>
  <div
    className="buttonContainer"
  >
    <InnerButton
      className="button teal disabled"
      onClick={[Function]}
      submit={false}
      withRouter={false}
    >
      <button
        className="button teal disabled"
        onClick={[Function]}
        type="button"
      >
        <div
          className="content"
        >
          Button
        </div>
      </button>
    </InnerButton>
  </div>
</Button>
`;
